google/cloud/security/common/gcp_api/admin_directory.py

```diff
@@ -35,22 +30,28 @@ class AdminDirectoryClient(_base_client.BaseClient):
         'https://www.googleapis.com/auth/admin.directory.group.readonly'
     ])
 
-    def __init__(self):
+    def __init__(self, global_configs):
         """Initialize.
 
-        Returns:
+        Args:
+            global_configs (dict): Global configurations.
         """
+
         super(AdminDirectoryClient, self).__init__(
-            credentials=self._build_credentials(),
+            global_configs,
+            credentials=self._build_credentials(global_configs),
             api_name=self.API_NAME)
 
         self.rate_limiter = RateLimiter(
-            FLAGS.max_admin_api_calls_per_day,
+            self.global_configs.get('max_admin_api_calls_per_day'),
```

Comment:
What do we do if this is None. Should we really have an exception here if the user flubbed a line in the config or something else? Yes, this is still me thinking about having defaults (outside the config) so that we don't have to fail if these aren't specified.